recipes/samtools/0.1.12/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+sed -i.bak 's/-lcurses/-lncurses/' Makefile
+sed -i.bak 's/-lz/-lz -ltinfo/' Makefile
+sed -i.bak 's/\(inline void __ks_insertsort\)/static \1/'  ksort.h
+export CFLAGS="-I$PREFIX/include"
```

Comment:
I think this should be 
```
export CFLAGS="$CFLAGS -I$PREFIX/include"
```
in case the CFLAGS should be default set to something in the future.